src/test/javascript/spec/service/programming-exercise-grading.service.spec.ts

```diff
@@ -123,12 +132,12 @@ describe('ProgrammingExerciseGradingService', () => {
             .pipe(tap((newTestCases) => (testCasesExercise1Subscriber2 = newTestCases)))
             .subscribe();
 
-        expect(testCasesExercise1Subscriber1).to.equal(testCases1);
-        expect(testCasesExercise1Subscriber2).to.equal(testCases1);
+        expect(testCasesExercise1Subscriber1).toBe(testCases1);
+        expect(testCasesExercise1Subscriber2).toBe(testCases1);
 
         gradingService.notifyTestCases(exercise1.id, newTestCasesOracle);
 
-        expect(testCasesExercise1Subscriber1).to.equal(newTestCasesOracle);
-        expect(testCasesExercise1Subscriber2).to.equal(newTestCasesOracle);
+        expect(testCasesExercise1Subscriber1).toBe(newTestCasesOracle);
+        expect(testCasesExercise1Subscriber2).toBe(newTestCasesOracle);
```

Comment:
Do you want to test that it has the same reference or the same content? If it is the latter, I would suggest using `toEqual()`